docs-client/src/containers/MethodPage/DebugPage.tsx

```diff
@@ -547,110 +564,132 @@ const DebugPage: React.FunctionComponent<Props> = ({
+                    </Grid>
+                  </Grid>
+                  <SyntaxHighlighter
+                    language="json"
+                    style={githubGist}
+                    wrapLines={false}
+                  >
+                    {debugResponse}
+                  </SyntaxHighlighter>
+                </Grid>
+              </Grid>
+              <Snackbar
+                open={snackbarOpen}
+                message={snackbarMessage}
+                autoHideDuration={3000}
+                onClose={dismissSnackbar}
+                action={
+                  <IconButton color="inherit" onClick={dismissSnackbar}>
+                    <CloseIcon />
+                  </IconButton>
+                }
               />
-            )}
-            <Typography variant="body2" paragraph />
+            </div>
+          </Section>
+        </DialogContent>
+        <DialogActions style={{ justifyContent: 'space-between' }}>
+          <div>
             <Button variant="contained" color="primary" onClick={onSubmit}>
               Submit
             </Button>
             <Button variant="text" color="secondary" onClick={onExport}>
               Copy as a curl command
             </Button>
-          </Grid>
-          <Grid item xs={12} sm={6}>
-            <Grid container spacing={1}>
-              <Grid item xs="auto">
-                <Tooltip title="Copy response">
-                  <div>
-                    <IconButton
-                      onClick={onCopy}
-                      disabled={debugResponse.length === 0}
-                    >
-                      <FileCopyIcon />
-                    </IconButton>
-                  </div>
-                </Tooltip>
-              </Grid>
-              <Grid item xs="auto">
-                <Tooltip title="Clear response">
-                  <div>
-                    <IconButton
-                      onClick={onClear}
-                      disabled={debugResponse.length === 0}
-                    >
-                      <DeleteSweepIcon />
-                    </IconButton>
-                  </div>
-                </Tooltip>
-              </Grid>
-            </Grid>
-            <SyntaxHighlighter
-              language="json"
-              style={githubGist}
-              wrapLines={false}
-            >
-              {debugResponse}
-            </SyntaxHighlighter>
-          </Grid>
-        </Grid>
-        <Snackbar
-          open={snackbarOpen}
-          message={snackbarMessage}
-          autoHideDuration={3000}
-          onClose={dismissSnackbar}
-          action={
-            <IconButton color="inherit" onClick={dismissSnackbar}>
-              <CloseIcon />
-            </IconButton>
-          }
-        />
-      </div>
-    </Section>
+          </div>
+          <Button
+            autoFocus
+            onClick={() => setDebugFormIsOpen(false)}
+            color="primary"
+          >
+            Close
+          </Button>
```

Comment:
I found it difficult to see the close button. How about adding color to the button so that users easily notice it?

Reply:
That makes sense, let me play with the colors a bit. I was thinking most people would actually click outside the focus area to close to the dialog that's why I didn't realize that.